espn_api/football/fantasy.py

```diff
@@ -0,0 +1,172 @@
+from espn_api.football import League
+from operator import attrgetter
+
+# Add award to dict of teams
+def award(team_name, award_string):
+	if awards.get(team_name) != None:
+		awards[team_name].append(award_string)
+	else: 
+		awards[team_name] = [award_string]
+
+# Print all awards for each team
+def printAwards(awards):
```

Comment:
It's a little confusing to me that this method takes awards as a param but the one above uses the global variable